### PR TITLE
Add return 'local' version for projects with no repo

### PR DESCRIPTION
**Pull request by t-pyrope**

### CODE REVIEW

**Review by Pe5h4 — CHANGES_REQUESTED** [review]
@t-pyrope Add this change to the CHANGELOG please

File: webpack/common.js [hunk 1]
```diff
@@ -58,5 +58,12 @@ exports.convertKeysForHtml = function(obj){
 }
 
 exports.getVersion = function(){
-	return execSync("git describe --abbrev=7 --tags --dirty=+dirty --always", { encoding: 'utf8' }).toString();
+	try {
+		const { stdout } = execSync("git describe --abbrev=7 --tags --dirty=+dirty --always", { encoding: 'utf8' }).toString();
+		return stdout;
+	} catch (e) {
+		console.log("Something has happend when trying to get version");
```
Comment (ateska) [inline webpack/common.js:65]: "Something has happend when trying to get version" ???
->
"Error when getting version from git"

**Review by ateska — CHANGES_REQUESTED** [review]
Please change the text to provide at least some kind of information.
Then mergeable.

File: webpack/common.js (hunk 1 above)
Comment (titkov49) [reply webpack/common.js:65]: I would also add in message something that it doesn't affect build process in order not to disorient user about build itself.

File: webpack/common.js [hunk 2]
```diff
@@ -58,5 +58,13 @@ exports.convertKeysForHtml = function(obj){
 }
 
 exports.getVersion = function(){
-	return execSync("git describe --abbrev=7 --tags --dirty=+dirty --always", { encoding: 'utf8' }).toString();
+	try {
+		const { stdout } = execSync("git describe --abbrev=7 --tags --dirty=+dirty --always", { encoding: 'utf8' }).toString();
+		return stdout;
+	} catch (e) {
+		console.log("Error when getting version from git");
+		console.log("This does not affect build process");
```
Comment (titkov49) [inline webpack/common.js:66]: This should be one line. Something like this:
`Error when getting version from git. This error doesn't affect build process but you won't be able to see current git version of the repository.`

**Review by titkov49 — CHANGES_REQUESTED** [review]

**Review by titkov49 — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -101,4 +101,6 @@
 
 - Remove unused packages and update packages with vulnerabilities (INDIGO Sprint 210625, [!135](https://github.com/TeskaLabs/asab-webui/pull/135))
 
-- Fix microservices containers indentation in order to keep clean code style (INDIGO Sprint 210723 ,[!143](https://github.com/TeskaLabs/asab-webui/pull/143))
\ No newline at end of file
+- Fix microservices containers indentation in order to keep clean code style (INDIGO Sprint 210723 ,[!143](https://github.com/TeskaLabs/asab-webui/pull/143))
+
+- Update `getVersion` function in `webpack/common.js` to prevent app from crashing when there is no repository for the project (INDIGO Sprint 210723, [!!146](https://github.com/TeskaLabs/asab-webui/pull/146))
```
Comment (Pe5h4) [inline CHANGELOG.md:106]: @t-pyrope Small remark - you have 2 exclamation marks in the changelog PR number

```
[!!146]
```

**Review by Pe5h4 — CHANGES_REQUESTED** [review]

**Review by ateska — APPROVED** [review]

**Review by Pe5h4 — APPROVED** [review]

**Pull request merged**